.github/workflows/ci.yaml

```diff
@@ -20,7 +20,10 @@ jobs:
         run: make lint
 
   setup:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.platform }}
```

Comment:
I'd suggest setting this aside for its own PR and discussion.

Reply:
Sure. I'll take care of it.